### PR TITLE
ci: Run ⊂ of rpm-ostree's vmcheck

### PR DESCRIPTION
**Pull request by cgwalters**
This is a quick hack to get us more than unit testing, albeit indirectly.

See: https://github.com/projectatomic/rpm-ostree/issues/662

### CODE REVIEW

**Comment by cgwalters** [comment]
bot, retest this please

**Comment by jlebon** [comment]
bot, retest this please

**Comment by jlebon** [comment]
test comment

**Comment by jlebon** [comment]
test comment

**Comment by jlebon** [comment]
bot, retest this please

**Comment by cgwalters** [comment]
So it did break the unit tests, but not vmcheck.  Hmm.

**Comment by cgwalters** [comment]
OK, this looks good to go now.

**Comment by cgwalters** [comment]
(Forgot to mention I'm resurrecting this PR from limbo)

File: ci/rpmostree.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+cd rpm-ostree
+build
+# We want to capture automake results
+cleanup() {
+    mv test-suite.log ${codedir} || true
+}
+trap cleanup EXIT
+make -j 8 check
+# Just a subset ⊂ for now to avoid CI overload
+#make vmcheck TESTS=layering-basic
```
Comment (jlebon) [inline ci/rpmostree.sh]: Let's at least run `test-basic` and/or `test-layering-basic`? Otherwise, there's no point in provisioning a host too, right?

Comment (cgwalters) [reply ci/rpmostree.sh]: Yeah the idea was to do the unit tests first.  I'll can take a stab at doing those two in this PR, or do you think we should do a separate PR?

Comment (jlebon) [reply ci/rpmostree.sh]: Doing it in a separate PR is cool with me! Though maybe let's drop the host provisioning in this PR then?

Comment (cgwalters) [reply ci/rpmostree.sh]: Eh let's do it in this one.  Working on it now.

**Comment by jlebon** [comment]
> ./tests/vmcheck/install.sh: line 32: rsync: command not found

**Comment by cgwalters** [comment]
Hmm, the fact that we failed to log in via ssh is worrying but OTOH the current rpm-ostree PRs are going in and I verified they were using `v2017.15-26-g854a823e`.  Plus local testing passes.

**Comment by cgwalters** [comment]
bot, retest this please

**Comment by cgwalters** [comment]
> (/tmp/tmp.lpaS9ZW5jm/rpm-ostree/.libs/lt-rpm-ostree status:27613): GLib-CRITICAL **: g_source_set_ready_time: assertion 'source->ref_count > 0' failed

Ooh, that one again.

**Comment by cgwalters** [comment]
bot, retest this please

**Comment by cgwalters** [comment]
> Jan 17 17:04:12 vmcheck rpm-ostree[1628]: /usr/bin/rpm-ostree: error while loading shared libraries: libsoup-2.4.so.1: cannot open shared object file: No such file or directory

Aaargh, of course.

**Comment by cgwalters** [comment]
> f27-rpmostree — All tests passed. 

:champagne: :clinking_glasses: :fireworks:

**Comment by cgwalters** [comment]
Rebased 🏄 and squashed 🗜.

**Comment by jlebon** [comment]
Whoooo, very nice!
@rh-atomic-bot r+ bf90ce3

**Comment by jlebon** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**